projects/js-packages/scan/package.json

```diff
@@ -0,0 +1,68 @@
+{
+	"private": false,
+	"name": "@automattic/jetpack-scan",
+	"version": "0.1.0-alpha",
+	"description": "A JS client for consuming Jetpack Scan services",
+	"homepage": "https://github.com/Automattic/jetpack/tree/HEAD/projects/js-packages/scan/#readme",
+	"bugs": {
+		"url": "https://github.com/Automattic/jetpack/labels/[JS Package] Scan"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Automattic/jetpack.git",
+		"directory": "projects/js-packages/scan"
+	},
+	"license": "GPL-2.0-or-later",
+	"author": "Automattic",
+	"scripts": {
+		"build": "pnpm run clean && pnpm run compile-ts",
+		"clean": "rm -rf build/",
+		"compile-ts": "tsc --pretty",
+		"test": "jest"
+	},
+	"type": "module",
+	"devDependencies": {
+		"@automattic/jetpack-webpack-config": "workspace:*",
+		"@babel/core": "7.23.5",
+		"@babel/plugin-transform-react-jsx": "7.23.4",
+		"@jest/globals": "29.7.0",
+		"@storybook/addon-actions": "7.4.6",
+		"@storybook/blocks": "7.4.6",
+		"@storybook/react": "7.4.6",
+		"@testing-library/react": "13.4.0",
+		"@types/jest": "29.5.10",
+		"@types/react": "18.2.33",
+		"@wordpress/babel-plugin-import-jsx-pragma": "4.30.0",
+		"babel-jest": "29.7.0",
+		"jest": "^29.7.0",
+		"jest-environment-jsdom": "29.7.0",
+		"typescript": "5.0.4"
+	},
+	"exports": {
+		".": {
+			"jetpack:src": "./src/index.ts",
+			"types": "./build/index.d.ts",
+			"default": "./build/index.js"
+		}
+	},
+	"main": "./build/index.js",
+	"types": "./build/index.d.ts",
+	"dependencies": {
+		"@automattic/jetpack-api": "workspace:*",
+		"@automattic/jetpack-base-styles": "workspace:*",
+		"@automattic/jetpack-connection": "workspace:*",
+		"@automattic/jetpack-shared-extension-utils": "workspace:*",
+		"@wordpress/api-fetch": "6.44.0",
+		"@wordpress/element": "5.24.0",
+		"@wordpress/i18n": "4.47.0",
+		"@wordpress/url": "3.48.0",
+		"debug": "4.3.4",
+		"react": "^18.2.0",
+		"react-dom": "^18.2.0"
+	},
+	"peerDependencies": {
+		"react": "^18.2.0",
+		"react-dom": "^18.2.0",
+		"@wordpress/i18n": "4.47.0"
```

Comment:
Why the peer dep on `@wordpress/i18n`?

Reply:
Peers will need to use `@wordpress/i18n` to use translatable strings provided by the package, a bit more info here: https://github.com/Automattic/jetpack/pull/34530/files#diff-265b9ba8c9fd1dbc6ab4c5f95c07260e3e4424ac4a08a2eb6a7cf98d735f0a01R29

That is the only reason I have listed it as a peer dep, does it make sense for this use case?

Reply:
Hmm. Is the use case here regarding if someone wants to use the package outside of a WordPress context?

Within WordPress things should just work, [@wordpress/dependency-extraction-webpack-plugin](https://www.npmjs.com/package/@wordpress/dependency-extraction-webpack-plugin) should externalize the reference to `@wordpress/i18n` already and WordPress should load the relevant i18n itself.